deploy/upgrade-pgo.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Get current working directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+NEWLINE=$'\n'
+
+echo "${NEWLINE}Before running the Postgres Operator upgrade script, please ensure you have already updated and"
+echo "sourced your user's .bashrc file, as well as your \$PGOROOT\\postgres-operator\\pgo.yaml configuration file."
+echo "${NEWLINE}More information can be found in the \"Default Installation - Configure Environment\" section"
+echo "of the Postgres Operator Bash installation instructions, located here:"
+echo "${NEWLINE}https://crunchydata.github.io/postgres-operator/stable/installation/operator-install/${NEWLINE}"
```

Comment:
I think it'd be cleaner just to use `echo ""`

```shell
echo ""
echo "Before running.."
echo "sourced your..."
echo ""
echo ""More information"
```
etc.

Reply:
works for me :+1: